ajax/libs/material-datetime-picker/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "material-datetime-picker",
+  "filename": "material-datetime-picker.min.js",
+  "version": "2.4.0",
+  "description": "A Vanilla JS Material Design date/time picker component",
+  "homepage": "https://ripjar.github.io/material-datetime-picker/",
+  "keywords": [
+    "material-design",
+    "date-picker",
+    "datetime-picker",
+    "javascript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ripjar/material-datetime-picker"
+  },
+  "author": "Joe Whitfield-Seed <joeseed86@gmail.com>",
```

Comment:
Let's add two authors.
One is [`Joe Whitfield-Seed <joeseed86@gmail.com>`](https://github.com/ripjar/material-datetime-picker/blob/master/bower.json#L6), another is [`jivings`](https://github.com/ripjar/material-datetime-picker/blob/master/package.json#L19)
Actually, the real author is jivings, you cam see him in commits log.
However, the author field of package.json is different from bower.json.
I think the best way is add both of them.

Many thanks.